settingshelper.py

```diff
@@ -172,12 +172,19 @@ def celery_failure_handler(task, exc, task_id, args, kwargs, einfo):
 
 
 def get_allowed_websocket_channels(request, channels):
+    from django.conf import settings
     from django.core.exceptions import PermissionDenied
+    from corehq.apps.users.models import CouchUser
     if request.user and request.user.is_authenticated() and request.user.is_superuser:
         return channels
     else:
+        couch_user = CouchUser.from_django_user(request.user)
```

Comment:
Can you not use `request.couch_user` which should have already been populated in the middleware?